python_modules/libraries/dagster-dbt/dagster_dbt/dagster_dbt_translator.py

```diff
@@ -106,6 +105,11 @@ def get_asset_spec(
         # memoize resolution for a given manifest & unique_id
         # since we recursively call get_asset_spec for dependencies
         memo_id = (id(manifest), unique_id)
+
+        # Don't initialize this in the constructor in case a subclass does not call __init__
+        if not hasattr(self, "_resolved_specs"):
+            self._resolved_specs = {}
```

Comment:
can't we just have 

```python
class DagsterDbtTranslator:
    ...
    _resolved_specs: dict[tuple, AssetSpec] = {}
```

instead of setting this in the constructor?

Reply:
Does that not make it shared across multiple instances?

Reply:
ah yeah fair enough